src/Microsoft.Health.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -220,7 +232,7 @@ public override void Set(SqlDataRecord record, int ordinal, decimal value)
     public class FloatColumn : Column<double>
     {
         public FloatColumn(string name, byte precision)
-            : base(name, SqlDbType.Float, false, precision, 0)
+            : base(name, SqlDbType.Float, false, ColumnUtilities.GetLengthForFloatColumn(precision), precision, 0, 0, SqlCompareOptions.None, null)
```

Comment:
Kind of surprising, but it looks like this constructor, when called with float, throws ignores maxLength and precision. Bizarre api!

Reply:
I mean the SqlMetadata contructor :)